src/routes/rules/RulesView.vue

```diff
@@ -1,289 +1,374 @@
+        <v-col class="sidebar-container" lg="3" sm="12">
+          <ul class="ms-5 sidebar-title mt-8">
+            <li
+              v-for="{ title, href, id } in sectionTitle"
+              :id="'listItem_' + id"
+              :key="title"
+              ref="items"
+            >
+              <button
+                :class="[
+                  activeTitle === id ? 'text-newPrimary' : 'text-surface-2',
+                  'text-h5 text-decoration-none',
+                ]"
+                @click="onClick(href)"
+              >
+                {{ t(title) }}
+              </button>
+            </li>
+          </ul>
         </v-col>
-      </v-row>
 
-      <!-- Royals -->
-      <v-row class="flex-column">
-        <div class="d-flex">
-          <v-icon
-            size="x-large"
-            color="black"
-            class="mr-4"
-            icon="mdi-crown"
-            aria-label="crown icon"
-            aria-hidden="false"
-            role="img"
+        <!-- Rules -->
+        <v-col>
+          <RulePreviewDialog
+            v-model="previewDialog"
+            :image-url="imageUrl"
+            :title="previewTitle"
+            @close="closeAnimate"
           />
-          <h1 class="gradient-text">
-            {{ t('rules.royals.title') }}
-          </h1>
-        </div>
-        <p class="d-block">
-          {{ t('rules.royals.text') }}
-        </p>
-      </v-row>
-      <v-row
-        v-for="(ruleRow, rowIndex) in royals"
-        :key="`royal-row-${rowIndex}`"
-        align="start"
-        class="my-6"
-      >
-        <v-col
-          v-for="(rule) in ruleRow"
-          :key="rule.title"
-          md="6"
-          sm="12"
-          class="my-4"
-        >
-          <RulePreview
-            ref="preview"
-            :title="t(rule.title)"
-            :description="t(rule.description)"
-            :animated-img="rule.animatedImg"
-            :static-img="rule.staticImg"
-            :icon="rule.icon"
-            @animate="handleAnimate"
-          />
-        </v-col>
-      </v-row>
+          <v-row
+            id="introduction"
+            v-intersect="{
+              handler: onIntersect,
+            }"
+            class="flex-column align-start"
+          >
+            <div class="mt-8">
+              <h1 class="text-h2 text-surface-2 mb-5 section-title">
+                {{ t('rules.introduction') }}
+              </h1>
+              <h1>
+                {{ t('rules.cuttleTitle') }}
+              </h1>
+              <p class="d-block">
+                {{ t('rules.cuttleText') }}
+              </p>
+            </div>
+          </v-row>
+          <!-- Tutorial -->
+          <v-row class="flex-column align-start mt-5">
+            <h1>
+              {{ t('rules.rulesTitle') }}
+            </h1>
+            <p>
+              {{ t('rules.rulesReadText') }}
+              <a href="/img/cuttle_rules.pdf" target="_blank" class="text-anchor"> Cuttle Cheestsheet </a>
```

Comment:
This is a typo in prod too
```suggestion
              <a href="/img/cuttle_rules.pdf" target="_blank" class="text-anchor"> Cuttle Cheastsheet </a>
```